### ISSUE TITLE
a11y compatibility

### ISSUE THREAD

**Issue by andy-schwedler** [andy-schwedler/capstone-project#29]
## Value Proposition

**As a** beaver with disabilities 

**I need** an accessible web page

**to be able to** use it

## Description

This story is about making this web app more accessible for people with disabilities. Therefore the goal should be to reach a Lighthouse Accessibility Score of 90+

## Acceptance Criteria
- Lighthouse Accessibility Score of 90+

## Tasks

- [x] add aria-labels to buttons
- [x] add aria-labels to links
- [x] add lang attribute to html element
- [ ] adjust header contrast ratio
- [ ] adjust footer contrast ratio
- [ ] adjust placeholder in search component

**Comment by BrainCodeJockey** [comment]
Hi Andy, a great US. is fine for me. : )
